lib/AdminPages/Pages/FormEditor.php

```diff
@@ -70,12 +191,12 @@ public function __construct(
   }
 
   public function render() {
-    if (!isset($_GET['id']) && !isset($_GET['action'])) {
+     if (!isset($_GET['id']) && !isset($_GET['action'])) {
       $this->renderTemplateSelection();
       return;
     }
-    if ($_GET['action'] === 'create') {
-      $this->createEmptyForm();
+    if (isset($_GET['action']) && $_GET['action'] === 'create') {
+      $this->createForm($_GET['template-id']);
```

Comment:
What if `template-id` is not present? This is changing in multiple commits so I'm a bit confused now.

Reply:
This code was removed. Sorry.